docs/en/03_Upgrading/03_Migrating_off_CWP.md

```diff
@@ -0,0 +1,44 @@
+
+The CWP CMS release has been discontinued. CWP project who wish to continue to receive update most migrate their code base off the latest CWP release back to the regular Silverstripe CMS release.
+
+## Steps to migcate away from CWP
+
+Before you began this process you must identify which version of Silverstripe CMS Recipe you want to upgrade to. You can upgrade to [Silverstripe CMS Recipe 4.9.0](/changelogs/4.9.0/) or greater.
+
+[Review the changelog list](/changelogs/) to choose the Silverstripe CMS Recipe version you wish to upgrade to. Read the changelog carefully. It may contain additional release specific upgrade guidance beyond the steps in this guide.
+
+1. If your current composer file contains `cwp/cwp-installer`, run `composer update-recipe cwp/cwp-installer` to inline the recipe.
+2. If your current composer file contains `cwp/cwp-recipe-cms`, run `composer update-recipe cwp/cwp-recipe-cms` to inline the recipe.
+3. If your current composer file contains `cwp/cwp-recipe-core`, replace it with `silverstripe/recipe-public-cloud`.
+4. If your current composer file contains `cwp/cwp-recipe-search`, replace it with `silverstripe/recipe-public-cloud-search`.
+5. Review all your composer requirements and update the constraints so the module versions documented in the changelog for you desired Silverstripe CMS Recipe release are installable.
+6. Run a `composer update` to get the lastest tag of each module.
+
+## Questions
+
+### What constraint should I use in my requirement version?
+
+That really depends on how comfortable your organisiation is with automatically applying updates to your Silverstripe CMS project.
+
+The three broad constraint categories you can use are, in descending order of restrictiveness:
+- _exact version_ which forces composer to install the exact version of a package (e.g.: `"silverstripe/framework: "4.9.0"`)
+- _tilde constraint_ which allows composer to install a later version of a package within the same minor release line (e.g.: `"silverstripe/framework: "~4.9.2"` will allow composer to install the `4.9.3` release, but not the `4.10.0` or `4.9.1` ones)
+- _caret constraint_ which allows composer to install a later version of a package within the same major release line (e.g.: `"silverstripe/framework: "^4.9.2"` will allow composer to install the `4.9.3` or `4.10.0` releases, but not the `5.0.0` one).
+
+The _tilde constraint_ is a suitable default and is the constraint style that ships in the `silverstripe/installer`.
+
+Read [Next Significant Release Operators](https://getcomposer.org/doc/articles/versions.md#next-significant-release-operators) in the composer documentation for more information version constraint operators.
+
+### What if I get a conflict when running composer update?
+
+### What happens if I do not migrate off CWP?
+
+### Is there a way to create a new project that would require all the modules a CWP project would have?
```

Comment:
I added a couple questions that I think might spring to mind here, but I would rather have some validations that we think those are usefull questions to answer before I spent too much time on them.